include/vkgcDefs.h

```diff
@@ -43,7 +43,7 @@
 #endif
 
 /// LLPC major interface version.
-#define LLPC_INTERFACE_MAJOR_VERSION 45
+#define LLPC_INTERFACE_MAJOR_VERSION 46
```

Comment:
@kuhar Does this version update need a special approval from AMD?

Reply:
We should wait for a review from AMD to confirm.